d2.config.js

```diff
@@ -0,0 +1,8 @@
+const config = {
+    type: 'app',
+    entryPoints: {
+        app: './src/components/App',
+    },
```

Comment:
`coreApp: true`
needs to be set for all apps that are bundled into the main DHIS2 WAR file.
`title` and `name` should also be set.

Reply:
Ah I didn't know that, thanks. Updated it in 19cdb39